matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -211,7 +234,7 @@ internal class LocalEchoEventFactory(private val credentials: Credentials, priva
     }
 
     private fun dummyEventId(roomId: String): String {
-        return roomId + "-" + dummyOriginServerTs()
+        return "m.${txNCounter++}"
```

Comment:
Is it unique enough?

Reply:
Not if you change the phone time in between, i used it to stay close to recommended implementation. I thought it could leak info about the client but as it's sent back only to caller i will move back to UUID.

Reply:
ok